pic-sure-auth-ui/src/main/webapp/admin/picSure/privilegeFunctions.js

```diff
@@ -0,0 +1,68 @@
+define(["util/notification"],
```

Comment:
Probably better to combine Role and Privilege operations into single file

Reply:
will probably create another ticket for genericFunctions later